doc-content/jbpm-docs/src/main/asciidoc/Integration/cdi-form-provider-service-con.adoc

```diff
@@ -0,0 +1,29 @@
+[id='cdi-form-provider-service-con_{context}']
+= The Form provider service service for CDI
+
+The `FormProviderService` service provides access to form representations, which are usually displayed on the user interface for both process forms and user task forms.
+
+The service relies on the concept of isolated form providers that can provide different capabilities and be backed by different technologies. The `FormProvider` interface describes the contract for implementations of form providers.
+
+.Definition of the `FormProvider` interface
+[source,java]
+----
+public interface FormProvider {
+
+    int getPriority();
+
+    String render(String name, ProcessDesc process, Map<String, Object> renderContext);
+
+    String render(String name, Task task, ProcessDesc process, Map<String, Object> renderContext);
+}
+----
```

Comment:
this has changed

public interface FormProviderService {

	String getFormDisplayTask(long taskId);
    
	String getFormDisplayProcess(String deploymentId, String processId);
}